packages/lib/plugin-connector/src/data-sources/get-sources.spec.js

```diff
@@ -87,4 +87,10 @@ describe('loadSourceOptions', () => {
 		expect('ssl' in result).toBeTruthy();
 		expect('rejectUnauthorized' in result.ssl).toBeTruthy();
 	});
+	it('should pickup boolean flags', () => {
+		vi.stubEnv('EVIDENCE_SOURCE__source__ssl__rejectUnauthorized', 'false');
+		const result = loadSourceOptions('source');
+		expect('ssl' in result).toBeTruthy();
+		expect('rejectUnauthorized' in result.ssl).toBeFalsy(); // <-- this is failing
+	});
 });
```

Comment:
`{ ssl: { rejectUnauthorized: 'false' } }`

Reply:
Apart from the value being a string i.e. 'false', there is another issue with this syntax :
- The `key in object` i.e. `'rejectUnauthorized' in result.ssl` syntax checks for the presence of a key in object. Its value is not being read / tested here.

All tests using this syntax are just checking the presence/absence of that key in object.